### ISSUE TITLE
Remove Asterisks from Restaurants Not Eaten at Yet

### ISSUE THREAD

**Issue by jfc3** [jfc3/10kapart#105]
Need to remove asterisks from restaurants not eaten at yet once #104 is completed and working. Will need to publish the updates to PROD server once they both are complete so we don't get duplicate sets of asterisks.

**Comment by jfc3** [comment]
My thinking is I should add a flag/field called "eaten" and if so add the word "yes" which will not show asterisks and if empty or anything else will get and asterisks placed after the restaurant's name and not part of the link. The asterisk will have aria-describedby added, so it will tell assistive technology users that these are places I have not yet eaten at yet.

**Comment by jfc3** [comment]
Need to work on and complete with issues #84 and #104.

**Comment by jfc3** [comment]
In preparation to add eaten here, photo, and ALT text code I'm adding the following fields for all these JSON files:

- eaten
- photo
- cptn

JSON City Files

- [ ] Annapolis, MD
- [ ] Atlanta, GA
- [ ] Austin, TX
- [ ] Baltimore, MD
- [ ] Boston, MA
- [ ] Buffalo, NY
- [ ] Cleveland, PA
- [ ] Columbus, OH
- [ ] Delaware
- [ ] Kentucky
- [ ] Las Vegas, NV
- [ ] Maryland
- [ ] Memphis, TN
- [ ] Mississippi
- [ ] Nashville, TN
- [ ] New Orleans, LA
- [ ] New York, NY
- [ ] Northern Virginia
- [ ] Ocean City, NJ
- [ ] Philadelphia, PA
- [ ] Portland, OR
- [ ] Rochester, NY
- [ ] San Diego, CA
- [ ] Syracuse, NY
- [ ] Washington, DC

**Comment by jfc3** [comment]
Being that this repository is the same as the [ATE HERE](https://github.com/jfc3/atehere) one only with smaller JSON file sizes to keep the total page load below 10K, I have stopped doing double the work.

If you want to see, the application working live go to [John F Croston III Ate Here](http://jfciii.com/atehere/).

There will be no further code updating at this time unless I try something crazy and don't want to mess with the live version.

Closing out all the open issues too.
